app/views/components/_document-list.html.erb

```diff
@@ -0,0 +1,50 @@
+<%
+  group_index ||= false
+  track_clicks = "data-module=track-click" if group_index
+  dimension28 = items.count.to_s
+%>
+<ol class="app-c-document-list" <%= track_clicks %>>
+  <% items.each_with_index do |link, link_index| %>
+    <li class="app-c-document-list__item">
+      <h3 class="app-c-document-list__item-title">
```

Comment:
@nickcolley my understanding is that a classname in BEM should have at most one instance of -- and one instance of __ i.e. block__element--modifier. Here app-c-document-list is the block, __item and __item-title are the elements. The BEM documentation isn't super clear but I think this link explains it: http://getbem.com/faq/#what-would-be-a-class-name-for-an-element-inside-another-element-block__el1__el2-